incubator/skopeo-sync/values.yaml

```diff
@@ -0,0 +1,48 @@
+## -- A secret containing all of the authfiles necessary
+secretName: registry-creds
+
+## -- The number of successful finished jobs to retain. Value must be non-negative integer
+successfulJobsHistoryLimit: 0
+
+## -- The number of failed finished jobs to retain. Value must be non-negative integer.
+failedJobsHistoryLimit: 1
```

Comment:
suggestion: put these together in a `jobOptions` object

Reply:
Since these are cronjob options, and this is the _only_ thing this chart makes, I feel like that's a little excessive. Especially since then we would also probably have to nest the resources options.